src/mango/test/mango.py

```diff
@@ -165,13 +166,13 @@ def list_indexes(self, limit="", skip=""):
 
     def delete_index(self, ddocid, name, idx_type="json"):
         path = ["_index", ddocid, idx_type, name]
-        r = self.sess.delete(self.path(path), params={"w":"3"})
+        r = self.sess.delete(self.path(path), params={"w": "1"})
         r.raise_for_status()
 
     def bulk_delete(self, docs):
         body = {
             "docids" : docs,
-            "w": 3
+            "w": 1
```

Comment:
eh not sure if I need to do this for bulk_delete since it's only index deletion is causing issues